styles.css

```diff
@@ -1 +1,84 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+
+/*styling*/
+body {background-color: #F0E9E4}
+
+/*styling*/
+section {
+    border-style: solid;
+    border-width: 2px;
+    border-radius: 5px;
+    border-color: #304D6D;
+    background-color: #82A0BC;
+    padding: 2px;
```

Comment:
Persoonlijk zou ik de padding iets groter maken bijvoorbeeld 10px. Is net wat leesbaarder